heltour/tournament/tests/test_views.py

```diff
@@ -93,7 +125,12 @@ def setUp(self):
         createCommonLeagueData()
 
     def test_template(self):
-        response = self.client.get(season_url('team', 'wallchart'))
+        # triggering a 404 writes to the log, disable that temporarily for nicer test output
```

Comment:
You don't have to fix this, but we should consider a nicer logging setup in general here.